scripts/translations.js

```diff
@@ -0,0 +1,69 @@
+const fs = require('fs');
+const path = require('path');
+
+const shell = require('shelljs');
+
+const main = async () => {
+  const {stdout} = shell.exec('git diff --name-only --cached', {silent: true});
+  const changes = stdout.trim().split('\n');
+
+  let packages;
+
+  try {
+    packages = fs.readdirSync(path.resolve(process.cwd(), 'packages'), 'utf8');
+
+    packages = packages.filter((pkg) => {
+      return pkg.indexOf('strapi') !== -1;
+    });
+  } catch (error) {
+    return console.error(`Can't get strapi packages`);
+  }
+
+  const packageCheckingPromises = packages.map((pkg) => {
+    return new Promise(async (resolve) => {
+      const translationsPath = path.resolve(process.cwd(), 'packages', pkg, 'admin', 'src', 'translations');
+
+      let files;
+
+      try {
+        files = fs.readdirSync(translationsPath, 'utf8');
+      } catch (error) {
+        return resolve();
+      }
+
+      const updateFilesPromises = files.map((file) => {
+        if (!changes.includes(path.join('packages', pkg, 'admin', 'src', 'translations', file))) {
```

Comment:
... Is this a secure validation? As I said above, what if the commit is done not in the root path? 🤔

Reply:
Never mind... I just realize that is already an assumption:
![capture](https://user-images.githubusercontent.com/5768813/47595632-f1a72000-d946-11e8-9b30-bb88598e212c.PNG)

The error is by the pre-commit hook `lint` that is searching for the `eslint` command as a relative path ... 😕

😢